frontend/package.json

```
@@ -106,6 +106,9 @@
     "node": ">=22"
   },
   "overrides": {
+    "vite": {
+      "rollup": "npm:@rollup/wasm-node"
```

```suggestion
      "rollup": "@rollup/wasm-node"
```

**Reason:** the `npm:` prefix isn't needed since we're already using npm.

---

https://github.com/vitejs/vite/issues/15167#issuecomment-1833274104

---

Works on my end.
Anyway, shouldn't you also add a version so we have reproducible builds?

---

version specified now